app/controllers/dashboard_controller.rb

```diff
@@ -19,6 +19,20 @@ def index
     redirect_to :action => 'show'
   end
 
+  def dialog_definition
+    plugin = Vmdb::Plugins.find { |plug| plug.name.chomp('::Engine') == params[:class] }
+    if plugin.present?
+      name = plugin.root.join('dialogs', params[:name] + '.json')
```

Comment:
so this looks dangerous, we can do something like ```params[:name] = "/../../../../.ssh/id_rsa"``` ?

Reply:
yes, please, see the WIP on top. This is not ready for review.